modules/tensor_mechanics/include/materials/ComputeReturnMappingStress.h

```diff
@@ -42,7 +42,7 @@ class ComputeReturnMappingStress : public ComputeFiniteStrainElasticStress
   const MaterialProperty<RankFourTensor> & _elasticity_tensor;
   const MaterialProperty<RankTwoTensor> & _strain_increment;
   MaterialProperty<RankTwoTensor> & _elastic_strain_old;
-  std::vector<RecomputeRadialReturn*> _models;
+  std::vector<MooseSharedPointer<RecomputeRadialReturn>> _models;
```

Comment:
Oh boy, it looks like this space is starting to get closed on the angle brackets! I've been avoiding that until now. @dschwen hasn't. Perhaps we need to decide what we'd like to do as a team and put it in the style guide. I know that our Emacs settings are going to need some adjustments. I've noticed that the syntax highlighting gets messed up when it encounters these so that's one of the reasons I don't like them.